packages/build/tests/install/snapshots/tests.mjs.md

```diff
@@ -1012,21 +1016,21 @@ Generated by [AVA](https://avajs.dev).
           package: '@netlify/plugin-local-install-core'␊
         - inputs: {}␊
           origin: config␊
-          package: ./plugin/main.js␊
+          package: ./plugin/main.cjs␊
     ␊
     > Context␊
       production␊
     ␊
     > Loading plugins␊
-       - ./plugin/main.js from netlify.toml␊
+       - ./plugin/main.cjs from netlify.toml␊
```

Comment:
Some of the files have to be kept as CommonJS and use the file extension `.cjs`, which explains those change in some of the test snapshots.